### DISCUSSION TITLE
how to deploy model on HUAWEI Ascend 310

### DISCUSSION THREAD

**Discussion by GreenAvocado92** [open-mmlab/mmdeploy#1264] [Q&A]
how to deploy model on HUAWEI Ascend 310

**Comment by lvhan028** [comment]
Hi, @GreenAvocado92 
I've given a talk about deploying OpenMMLab models on Huawei Ascend 310. 
You can find the talk [here](https://www.hiascend.com/forum/thread-0227104382781997002-1-1.html)
I'll catch up on the user guide in mmdeploy documents when I am available
